charts/corda/values.schema.json

```
@@ -0,0 +1,2574 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "http://example.com/example.json",
```

Doesn't really matter what this is for now but probably shouldn't be example.com. How about:
```suggestion
    "$id": "http://corda.net/corda-helm.json",
```